### PR TITLE
build: Bump version to 3.0.5 for the next release.

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -34,7 +34,7 @@ subprojects {
     apply plugin: 'findbugs'
 
     group = 'io.minio'
-    version = '3.0.4'
+    version = '3.0.5'
```
Comment (deekoder) [inline build.gradle]: @harshavardhana  - is this a proactive PR for the next release?

Comment (harshavardhana) [inline build.gradle]: Yes it is basically since we made a release, we move the current master to the next.

Comment (balamurugana) [inline build.gradle]: Basically this PR makes devel build (3.0.5-DEV) to differentiate current release 3.0.4, else devel build will conflict with current release.

**Comment by deekoder** [comment]
Got it.

**Comment by balamurugana** [comment]
@deekoder you could remove `Request changes`

**Pull request merged**